src/main/java/io/supertokens/useridmapping/TenantIdentifierStorageAndUserIdMapping.java

```diff
@@ -0,0 +1,44 @@
+/*
+ *    Copyright (c) 2023, VRAI Labs and/or its affiliates. All rights reserved.
```

Comment:
this should go in root level alongside Main.java? cause it's not a thing specific to useridmapping anymore

Reply:
Done